src/main/java/com/iridium/iridiumskyblock/commands/ExtraValueCommand.java

```diff
@@ -0,0 +1,96 @@
+package com.iridium.iridiumskyblock.commands;
+
+import com.iridium.iridiumcore.utils.StringUtils;
+import com.iridium.iridiumskyblock.IridiumSkyblock;
+import com.iridium.iridiumskyblock.database.Island;
+import com.iridium.iridiumskyblock.database.User;
+import com.iridium.iridiumskyblock.utils.PlayerUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.time.Duration;
+import java.util.*;
+
+public class ExtraValueCommand extends Command {
+
+    /**
+     * The default constructor.
+     */
+    public ExtraValueCommand() {
+        super(Collections.singletonList("extravalue"), "Set extra values of island", "%prefix% &7/is extravalue <add/remove/set> <player> <amount>", "iridiumskyblock.extravalue", false, Duration.ZERO);
+    }
+
+    /**
+     * Executes the command for the specified {@link CommandSender} with the provided arguments.
+     * Not called when the command execution was invalid (no permission, no player or command disabled).
+     * Gives extra values to specific player's island
+     *
+     * @param sender The CommandSender which executes this command
+     * @param args   The arguments used with this command. They contain the sub-command
+     */
+    @Override
+    public boolean execute(CommandSender sender, String[] args) {
+        if (args.length < 4) {
+            sender.sendMessage(StringUtils.color(syntax.replace("%prefix%", IridiumSkyblock.getInstance().getConfiguration().prefix)));
+            return false;
+        }
+
+        Player player = Bukkit.getPlayer(args[2]);
+        if (player != null) {
+            User user = IridiumSkyblock.getInstance().getUserManager().getUser(player);
+            Optional<Island> island = user.getIsland();
+            if (island.isPresent()) {
+                try {
+                    double amount = Double.parseDouble(args[3]);
+                    switch (args[1].toUpperCase(Locale.ENGLISH)) {
+                        case "ADD":
```

Comment:
Nevermind, thats dum

Reply:
I was about to say the same... But I don't know, it would be a very small subcommand. 

I can do a subcommand PR soon that adds subcommands for the other commands too.